app/views/account/tenants/show.html.slim

```diff
@@ -0,0 +1,15 @@
+h1= @tenant.name
+.btn-group
+  = link_to  "Edit", edit_account_tenant_path, class: 'btn btn-primary'
+  = link_to "Delete", account_tenant_path, method: :delete,
+                      data: { confirm: 'Are you sure you want to delete this tenant?' },
+                      class: 'btn btn-primary'
+.tenant
+  p
```

Comment:
this is something what must be improved in future. such HTML to show details is bad.

Reply:
I agree. I will improve this code in the future